README.md

```diff
@@ -12,12 +12,12 @@ The OSP Director Operator creates a set of Custom Resource Definitions on top of
 
 Hardware Provisioning CRDs
 --------------------------
-- openstacknetattachment: manages NodeNetworkConfigurationPolicy and NodeSriovConfigurationPolicy used to attach networks to virtual machines
+- openstacknetattachment: (internal) manages NodeNetworkConfigurationPolicy and NodeSriovConfigurationPolicy used to attach networks to virtual machines
 - openstacknetconfig: high level CRD to specify openstacknetattachments and openstacknets to describe the full network configuration. The set of reserved IP/MAC addresses per node are reflected in the status.
 - openstackbaremetalset: create sets of baremetal hosts for a specific TripleO role (Compute, Storage, etc.)
 - openstackcontrolplane: A CRD used to create the OpenStack control plane and manage associated openstackvmsets
 - openstacknet: Create networks which are used to assign IPs to the vmset and baremetalset resources below
-- openstackipset: Contains a set of IPs for a given network and role. Used internally to manage IP addresses.
+- openstackipset: (internal) Contains a set of IPs for a given network and role. Used internally to manage IP addresses.
 - openstackprovisionservers: used to serve custom images for baremetal provisioning with Metal3
 - openstackvmset: create sets of VMs using OpenShift Virtualization for a specific TripleO role (Controller, Database, NetworkController, etc.)
```

Comment:
should also add internal tag?

Reply:
openstackvmset as well?

Reply:
@stuggi I thought we said vmsets could be created separately? I guess if we make them internal the only way to create them is inside of the OSControlplane CR's then

Reply:
you usually don't have to create them separate, but sure we can leave them as non internal